common/src/main/scala/net/psforever/types/BailType.scala

```diff
@@ -8,7 +8,13 @@ object BailType extends Enumeration {
   type Type = Value
 
   val Normal = Value(0)
+  val Unk1   = Value(1) // to have Xtoolspar working
```

Comment:
Do we have a link to this tool?

Reply:
Which version do you want?  Sounours wrote the original (`XTools`) and I think between Sounours, Mazo, and myself, we each made unshared modifications to it for our own purposes.

It's a packet capture decoder.

Reply:
I was just wondering if it was on Github